contrib/craftscripts/draw.js

```diff
@@ -133,12 +133,12 @@ context.checkArgs(1, 3, "<image> <orientation> <palette>");
 
 var f = context.getSafeOpenFile("drawings", argv[1], "png", ["png", "jpg", "jpeg", "bmp"]);
 var sess = context.remember();
-var upright = argv[2] === "v";
+var upright = argv[2] == "v";
```

Comment:
All of these changes shouldn't be doing anything at all?

argv is a string array, and it's being compared to a string. In fact, these are all strings, so there is no case when it should fail on valid input.

Reply:
I don't know !
Nothing happens when I use "v" "opt" and "optHD"
Then I replaced all "===" by "==", and now it works.

Reply:
While I agree that this shouldn't matter, I can reproduce sooo I guess?
for reference, `typeof(argv[3])` returns `object` (as does `typeof(argv)`) despite it being passed in as `String[]` from java (and `Context.javaToJS` correctly returns a NativeJavaArray for it).

unsure if this is something that changed at some point in rhino, since this definitely used to work? (using 1.7.11 now since octy added that for forge, while we used to use 1.7R2)